SiennaOpenAPIModels.jl/Project.toml

```diff
@@ -6,8 +6,12 @@ version = "0.1.0"
 [deps]
 OpenAPI = "d5e62ea6-ddf3-4d43-8e4c-ad5e6c8bfd7d"
 PowerSystems = "bcd98974-b02a-5e2f-9ee0-a103f5c450dd"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Can you install Revise and TestEnv in your main environment and not the project dependencies?

https://github.com/JuliaTesting/TestEnv.jl?tab=readme-ov-file#note-on-installation